Cisco/cisco-esa/tests/test_ingest_log5.json

```diff
@@ -55,7 +55,14 @@
         "url": [
           "https://facebook.com/u/john.doe",
           "https://tiktok.com",
+          "https://tinyurl.es/tbdra",
           "www.twitter.com"
+        ],
+        "url_domain": [
+          "",
```

Comment:
I believe this one shouldn't be empty

Reply:
Seems that parser doesn't work properly for URLs without "https://" ("www.twitter.com" in this case)